models/maat/R/model2netcdf.MAAT.R

```diff
@@ -91,7 +88,7 @@ model2netcdf.MAAT <- function(outdir, sitelat = -999, sitelon = -999, start_date
     ############ Variable Conversions 
     ### Conversion factor for umol C -> kg C
     Mc <- 12.017  # molar mass of C, g/mol
-    umol2kg_C <- Mc * udunits2::ud.convert(1, "umol", "mol") * ud.convert(1, "g", "kg")
+    umol2kg_C <- Mc * udunits2::ud.convert(1, "umol", "mol") * udunits2::ud.convert(1, "g", "kg")
```

Comment:
can you not do this conversion in one call to ud.convert? "umol g" to "mol kg".  Also, those seem like odd units. Finally, check out the ```misc.convert``` function in utils that @istfer wrote -- it's a wrapper around ud.convert that tries to sort out other conversions, such at the molar mass conversion you've hacked into this.  Similar code is in model2netcdf.MAAT